app/views/feeds/_form_overrides.html.erb

```diff
@@ -6,6 +6,46 @@
 
     <div class="card-body">
       <div class="row">
+        <div class="col-12 mb-4">
+          <div class="form-floating input-group">
+            <%= form.text_field :subtitle %>
+            <%= form.label :subtitle %>
+            <%= field_help_text t(".help.subtitle") %>
+          </div>
+        </div>
+
+        <div class="col-12 mb-4">
+          <div class="form-floating input-group">
+            <%= form.trix_editor :description %>
+            <%= form.label :description %>
+            <%= field_help_text t(".help.description") %>
+          </div>
+        </div>
+
+        <div class="col-lg-6 mb-4">
+          <div class="form-floating input-group" data-controller="dynamic-form">
+            <%# HACK: display nested validation errors %>
```

Comment:
are these errors nested because they are technically on the `itunes_category` and not the `feed`? and this is not true of the `itunes_subcategory` below which is why that doesn't need any special error treatment?

Reply:
Yep - the validation is on the nested model, not on the Feed.  There's probably a cleaner way to accomplish this, like moving the validator up a level?  But I left this logic alone for now.

So there's nothing on the subcategory mostly because I'm lazy, and you can't make the UI give you a validation error on subcategory (the dropdown only has valid options, and subcat isn't required).  The only validation error you can get via the UI is that Category can't be blank.